lib/generators/decidim/templates/initializer.rb

```diff
@@ -3,4 +3,5 @@
   config.application_name = "My Application Name"
   config.mailer_sender    = "change-me@domain.org"
   config.authorization_handlers = [ExampleAuthorizationHandler]
+  # config.available_locales = %{en ca es}
```

Comment:
This is commented because it's a template - we're hinting the user that this is allowed.